src/features/singleRight/delegate/ChooseServicePage/ChooseServicePage.tsx

```diff
@@ -101,12 +102,12 @@ export const ChooseServicePage = () => {
                     onAdd={onAdd}
                     onUndo={onRemove}
                   />
-                  <GroupElements>
+                  <div className={classes.navigationContainer}>
```

Comment:
I can only see that the new NavigationSection component is used on the (so far unused) request version of this page. Is there a reason why it's not used here as well?